<!-- -->
<launch>
  <!-- Params -->
  <rosparam command="load" file="$(find speed_profile_planner)/config/params.yaml" />

  <node pkg="speed_profile_planner" type="speed_profile_planner" name="speed_profile_planner" output="screen">
    <remap from="/current_pose" to="/casper_auto/current_pose" />
    <remap from="/current_speed" to="/casper_auto/current_speed" />
    <remap from="/final_path" to="/casper_auto/final_path" />
    <remap from="/global_route" to="/casper_auto/global_route" />
    <remap from="/prediction/vehicles" to="/casper_auto/prediction/vehicles" />
    <remap from="/prediction/pedestrians" to="/casper_auto/prediction/pedestrians" />
    <remap from="/stopsign_marker" to="/casper_auto/odr_map/stoplines_marker" />
    <remap from="/crosswalk_marker" to="/casper_auto/odr_map/crosswalks_marker" />

    <remap from="/final_waypoints" to="/casper_auto/final_waypoints" />
    <remap from="/ghost_ahead_polygon" to="/casper_auto/ghost_ahead_polygon" />
  </node>
</launch>
